e2e/test/scenarios/embedding/embedding-reproductions.cy.spec.js

```diff
@@ -1317,3 +1317,134 @@ describe("issue 50373", () => {
     H.visitEmbeddedPage({ resource: { dashboard: ORDERS_DASHBOARD_ID } });
   });
 });
+
+describe("issue 51934 (EMB-189)", () => {
```

Comment:
I suggest trying to run this test and put `cy.pause()` in different steps, to see what it's doing. I've also added `cy.log` in many places that should explain what the test is performing.